Adyen/Constants/ClientConfig.cs

```diff
@@ -61,6 +61,9 @@ public class ClientConfig
         public static string StoredValueEndpointTest = "https://pal-test.adyen.com/pal/servlet/StoredValue";
         public static string StoredValueEndpointLive = "https://pal-live.adyen.com/pal/servlet/StoredValue";
         public static string StoredValueVersion = "v46";
+        public static string BalancePlatformEndpointTest = "https://balanceplatform-api-test.adyen.com/bcl/";
+        public static string BalancePlatformEndpointLive = "https://balanceplatform-api-live.adyen.com/bcl/";
+        public static string BalancePlatformVersion = "v2";
```

Comment:
are we using v2 or v1 to use the x-methodName?

Reply:
We can't use v1, it's completely messed up.